static/templates/message_reaction.handlebars

```diff
@@ -0,0 +1,4 @@
+<div class="message_reaction" title="{{this.title}}" data-emoji-name="{{this.emoji_name}}">
+  <div class="emoji emoji-{{this.emoji_name}}" />
+  <div class="message_reaction_count">{{this.count}}</div>
+</div>
```

Comment:
Can this just be inlined into message_reactions.handlebars?  Seems like we don't need more little files to manage this...